### DISCUSSION TITLE
Introduce a darker shade in the color scales for dark mode

### DISCUSSION THREAD

**Discussion by psirenny** [radix-ui/colors#67] [Ideas]
TailwindCSS recently added an additional `950` shade that works really well for dark mode. It allows for a gorgeous dark background without falling back to **black**. The [TailwindUI Commit Template](https://commit.tailwindui.com/) shows it off nicely:

<img width="1512" alt="Screenshot 2023-07-10 at 6 16 01 AM" src="https://github.com/radix-ui/colors/assets/463178/e8cbe494-9a6a-4613-99aa-9ca7155959b6">

To maintain parity, the light color scales could also add an additional lighter background shade. Indeed, many applications using Radix Colors already map the app background color to white, suggesting a desire for such a step.

**Comment by lumosminima** [comment]
Just ran into this I use color-shade-2 for body and bg-white for container and this gives a nice and calm light mode but doing the same in dark mode, bg-black ends up being too tonally different to the 12th shade. I follow darkest-to-light for UI layers in dark mode so if I use 12th shade as the blackest then the third UI layer (component cards) ends up being too bright. There really needs to be a more blacker shade for a more harmonious dark mode or at least give the formula (if there is one) for calculating more shades by ourselves.
